examples/decoding/plot_linear_model_patterns.py

```diff
@@ -45,7 +46,7 @@
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(2, None, method='iir')  # replace baselining with high-pass
+raw.filter(2, 15)  # replace baselining with high-pass
```

Comment:
It fixes a warning that this example generates about deprecation causing aliasing